pkg/processor/runtime/python/py/nuclio_sdk/event.py

```diff
@@ -58,13 +58,18 @@ def __init__(self,
         self.version = version
 
     def to_json(self):
-        return json.dumps(self, default=lambda o: o.__dict__)
+        obj = vars(self).copy()
```

Comment:
Why? I imagine this affects performance

Reply:
It's actually a bit faster:

New version
```
In [1]: from event import Event
In [2]: e = Event()
In [5]: %timeit e.to_json()
4.8 µs ± 9.46 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Old version
```
In [1]: from event import Event
In [2]: e = Event()
In [3]: %timeit e.to_json()
6.15 µs ± 22 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

However the main reason was that `make test-python` failed :)